### DISCUSSION TITLE
Can GacUI support VS2017?

### DISCUSSION THREAD

**Discussion by georgehandrew** [vczh-libraries/Release#18] [Q&A]
Hi Developer,
I have tried to install GacUI 1.1.3.0 through vcpkg for VS2017.
Although from the tutorial, it says'For Windows, Install Visual Studio 2017 Community or newer version', but seems all the tutorial projects have been built on vs2019. After I tried to down grade to toolset v141 from v142, it seems still need stdc++20.
Does GacUI really need any c++20 features?

Thanks!

**Comment by vczh** [comment]
Oh sorry to confuse you, because of some "technical issue", the document is staying at 1.0. I pointed it out here: http://vczh-libraries.github.io/document.html

Releases after 1.0 introduces stdc++20, and now I am using VS2022. VS2019 should be fine.

But if the only issue is about IDE, you could use the new msbuild with it. IDE the msbuild doesn't actually hardcoded together.
